docs/modules/release-notes/pages/all-releases/4_13_3S.adoc

```diff
@@ -0,0 +1,107 @@
+= Kobiton 4.13.3S release notes
+:navtitle: Kobiton 4.13.3S release notes
+
+_July 19, 2024_
+
+== New OS support
+
+kobiton 4.13.3S add support for the following:
+
+* iOS/iPadOS 18 Beta
+* Xcode 16 Beta
+
+== CoreDevice: Lightning mode support for iOS 17.4 devices and later without networking changes
+
+When Apple introduced some major changes with the rollout of iOS 17, it requires significant network configuration to support Lightning mode with Kobiton. The additional network requirements are hard to implement and sometimes pose a security risks for some organizations.
+
+With this release, we are happy to introduce CoreDevice, which is a solution to the above problem. With CoreDevice, iOS 17.4 and later devices can have Lightning mode without the additional network requirements.
+
+With iOS 17.0 to 17.3 devices, the necessary components to support CoreDevice is not available. We recommend hosting these devices in Standard mode with the additional Mac mini instead.

```

Comment:
Replace above with: 
Apple's CoreDevice cannot be used with iOS 17.0-17.3 devices. These devices must be hosted on a separate Mac mini in Standard mode.

Reply:
Replaced